build-system/tasks/extension-helpers.js

```diff
@@ -561,18 +563,54 @@ async function buildExtensionJs(path, name, version, latestVersion, options) {
   }
 
   if (name === 'amp-script') {
-    // Copy @ampproject/worker-dom/dist/amp/worker/worker.js to dist/ folder.
-    const dir = 'node_modules/@ampproject/worker-dom/dist/amp/worker/';
-    const file = `dist/v0/amp-script-worker-${version}`;
-    // The "js" output is minified and transpiled to ES5.
-    fs.copyFileSync(dir + 'worker.js', `${file}.js`);
-    // The "mjs" output is unminified ES6 and has debugging flags enabled.
-    fs.copyFileSync(dir + 'worker.mjs', `${file}.max.js`);
-
-    // Same as above but for the nodom worker variant.
-    const noDomFile = `dist/v0/amp-script-worker-nodom-${version}`;
-    fs.copyFileSync(dir + 'worker.nodom.js', `${noDomFile}.js`);
-    fs.copyFileSync(dir + 'worker.nodom.mjs', `${noDomFile}.max.js`);
+    const workerDomDir = 'node_modules/@ampproject/worker-dom';
+    const newDirExists = fs.existsSync(`${workerDomDir}/dist/amp-production`);
+    const targetDir = 'dist/v0';
+
+    log('Copying required @ampproject/worker-dom files.');
```

Comment:
Nit: Use the established logging standard in our compilation code. For example:

https://github.com/ampproject/amphtml/blob/b5dea36e0b8bd012585d50839766a084f99a3685/build-system/tasks/extension-helpers.js#L247-L250

Reply:
done.